_getting-started/index.md

```diff
@@ -7,55 +7,117 @@ includeTOC: true
 
 redirect_from: "/getting-started.html"
 ---
+## Try Scala without installing anything 
+You can try Scala, with access to all Scala compilers and 
+all published libraries with [Scastie](https://scastie.scala-lang.org/)
+
+##  Installing Scala
+### Using the Scala Installer (recommended way)
+The Scala Installer based on coursier will ensure that a JVM and standard Scala tools are installed on your system.  
+
+* Download coursier launcher and execute `setup` command
+
+<div class="main-download">
+    <div id="download-step-one">
+        <p>Follow <a href="https://get-coursier.io/docs/cli-overview.html#install-native-launcher" target="_blank">the instructions to install the <code>cs</code> launcher</a> then run:</p>
+        <p><code>$ ./cs setup</code></p>
+    </div>
+</div>
+
+
+Along with managing JVMs, it also install useful command-line tools: 
+Ammonite, coursier, scala, scalac, sbt-launcher, scalafmt.
+
+For more information, read [couriser-cli documentation](https://get-coursier.io/docs/cli-overview)
+
+### ...Or manually
+1. if you don't have java 1.8 or 11 installed, download 
+Java from [Oracle Java 8](https://www.oracle.com/java/technologies/javase-jdk8-downloads.html), [
+Oracle Java 11](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html), 
+or [AdoptOpenJDK 8/11](https://adoptopenjdk.net/). Refer JDK Compatibility for Scala/Java compatibility detail.
+1. Install [sbt](https://www.scala-sbt.org/download.html)
+
+
+## Create a Hello-world project with sbt
```

Comment:
Thinking more about this section, I’m actually on the fence.

I like the variety of small tutorials of the current website. They seem to address a broader audience. Here, we assume/require that people are familiar with using the command-line.

Note that for people who are not used to start a terminal and call sbt from there, it is also possible to create a Scala project using g8 templates from within IntelliJ:

![Screenshot from 2020-03-30 13-39-54](https://user-images.githubusercontent.com/332812/77908952-be2f9a00-728c-11ea-89f7-4fb36320f501.png)

![Screenshot from 2020-03-30 13-40-05](https://user-images.githubusercontent.com/332812/77908953-be2f9a00-728c-11ea-9c85-f8cc95602652.png)

But I see that I’m pulling too many things…! Maybe a simple solution could be to re-add the links to the other tutorials.